oxidation/parsec_cli/parsec_sequestre_dump/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
you may want to add a description for `clap` to use it in the `about` message

Reply:
The description will be the same for the package and the `about` of the cli, 
so it would be simpler to declare it once in the manifest

Reply:
Do you have any suggestion about the message ? I don't know what to say :sweat_smile:

Reply:
```
Extract encrypted files from a parsec export
```